src/com/oltpbenchmark/util/ScriptRunner.java

```diff
@@ -24,7 +24,6 @@
 import java.io.InputStreamReader;
```

Comment:
Maybe we remove this as well?
And create tables the same way we load the tables using the connection object?

But thats for a different time. :D

Reply:
agreed, though!